content/article/porting-a-dancer-plugin-to-dancer2.md

```diff
@@ -0,0 +1,177 @@
+  {
+    "title"       : "Porting a Dancer plugin to Dancer2",
+    "authors"     : ["christopher-white"],
+    "date"        : "2020-01-10T13:37:00",
+    "tags"        : ['dancer','dancer2','perl','perl-5','perl-programming','web-development','porting','plugin'],
+    "draft"       : true,
+    "image"       : "",
+    "thumbnail"   : "",
+    "description" : "How I ported a plugin without having to know it all first",
+    "categories"  : "development"
+  }
+
+----------
```

Comment:
We usually don't use line breaks here - if you're set on having one it's fine by me though.

Reply:
Based on your recent https://github.com/tpf/perldotcom/commit/7e498d266efc9cbbabdbfb5fba3a4975f92d405a , I think you mean line 13 should be deleted.  Am I understanding you correctly?  If so, no problem!

Reply:
Yes, thanks :+1: